packages/@react-stately/tooltip/src/useTooltipTriggerState.ts

```diff
@@ -81,11 +82,11 @@ export function useTooltipTriggerState(props: TooltipTriggerProps = {}): Tooltip
     if (immediate || closeDelay <= 0) {
       clearTimeout(closeTimeout.current);
       closeTimeout.current = null;
-      close();
+      closeCallback.current();
     } else if (!closeTimeout.current) {
       closeTimeout.current = setTimeout(() => {
         closeTimeout.current = null;
-        close();
+        closeCallback.current();
```

Comment:
no need to change this line of code (it has no bug), but changed it for consistency.
**use close callback only from ref**